.github/workflows/build-with-kokkos.yml

```diff
@@ -19,10 +19,10 @@ jobs:
           - image: nvidia/cuda:12.2.0-devel-ubuntu22.04
             preset: Cuda
             compiler: {cpp: g++-12, c: gcc-12}
-          - image: rocm/dev-ubuntu-22.04:5.4
+          - image: rocm/dev-ubuntu-22.04:5.7
```

Comment:
Why do we want to bump that version?

Reply:
Just a habit of removing the oldest version in such a case :) We can keep the 5.4 release too. Just noting that on LUMI (the European supercomputer ~ Frontier), the ROCm 5.4 stack is no longer available.

Reply:
OK. No problem. Let's keep 5.4.